Remotion/ObjectBinding/Sample/ReflectionBusinessObjectWebUIService.cs

```diff
@@ -35,6 +35,13 @@ public ToolTipBasedHelpInfo (string toolTip)
       }
     }
 
+    private readonly string _basePath;
+
+    public ReflectionBusinessObjectWebUIService (string basePath)
```

Comment:
We should be able to solve this using the resourceurlfactory.
Inject the ResourceUrlFactory, use
```
var targetType = ((BindableObjectClass) obj.BusinessObjectClass).TargetType;
x.GetResourceUrl (targetType, ResourceType.Image, targetType + ".gif");

Reply:
Yes but in one project the files are not yet at the correct location since the ObjectBinding.Web.Test project has not been converted yet. This change can be undone in once the other project has been converted.

It would also be possible to support both explicitly providing a path and defaulting to the resource path

Reply:
Ah, yes. option 2, then, please. And a comment that his should be cleaned up when the other project is done.
> It would also be possible to support both explicitly providing a path and defaulting to the resource path